pkg/utils/test/testUtils.go

```diff
@@ -72,6 +72,114 @@ var (
 	}
 )
 
+var (
+	MockWorkspaceWithDeleteOldCR = v1alpha1.Workspace{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "testWorkspace",
+			Namespace: "kaito",
+			Annotations: map[string]string{
+				"workspace.kaito.io/revision": "DeleteOldCR",
```

Comment:
I don't understand why the value of "workspace.kaito.io/revision" is not a hash value?